### PR TITLE
Add support for font-variant-ligatures

### PR DESCRIPTION
**Pull request by jpamental**
I'm hoping to add more tests for OpenType features. Test case support page is here:

https://jpamental.github.io/css-examples/font-features/font-variant-ligatures.html

If the feature is supported correctly, the left example should match the right (which is done with the lower-level syntax font-feature-settings)

### CODE REVIEW

File: features-json/font-variant-ligatures.json [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+  "usage_perc_y":0,
+  "usage_perc_a":0,
+  "ucprefix":false,
+  "parent":"",
+  "keywords":"font-feature,font-feature-settings,font-variant",
+  "ie_id":"",
+  "chrome_id":"",
+  "firefox_id":"",
+  "webkit_id":"",
+  "shown":true
```
Comment (HM100) [inline features-json/font-variant-ligatures.json]: Too many unknown support. Please change this line to “shown”:false

Comment (jpamental) [reply features-json/font-variant-ligatures.json]: @HM100 Is it the previous version support, or number of different browsers that's an issue? Just wondering how I can help this along as I'd really like to get more of the font features included.

Comment (HM100) [reply features-json/font-variant-ligatures.json]: The number of previous versions support is almost of unknown support excluding IE and Edge

Comment (jpamental) [reply features-json/font-variant-ligatures.json]: I've updated as requested

Comment (jpamental) [reply features-json/font-variant-ligatures.json]: I found a bunch more detail on when the feature shipped in almost all of the browsers, so updated with that info and set back to shown: true (please let me know if that's a problem and I'll update again)

Comment (HM100) [reply features-json/font-variant-ligatures.json]: and please fix those `x` to `y` or `y x` if those versions with `x` are prefixed supported

Comment (HM100) [reply features-json/font-variant-ligatures.json]: @jpamental Any ETA on fixing the issues?

**Comment by Fyrd** [comment]
Closing this PR as it's already covered by https://caniuse.com/#feat=mdn-css_properties_font-variant-ligatures and doesn't add a significant amount of new info.

**Pull request closed without merging**